__tests__/components/PopulationCalculation.test.tsx

```diff
@@ -0,0 +1,141 @@
+import '@testing-library/jest-dom';
+import PopulationCalculation from '../../components/PopulationCalculation';
+import { measureBundleState } from '../../state/atoms/measureBundle';
+import { patientTestCaseState } from '../../state/atoms/patientTestCase';
+import { getMockRecoilState, mantineRecoilWrap } from '../helpers/testHelpers';
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { Calculator } from 'fqm-execution';
+import MeasureUpload from '../../components/MeasureUpload';
+import PatientCreation from '../../components/ResourceCreation/PatientCreation';
+
+const MOCK_MEASURE_REPORT: fhir4.MeasureReport = {
+  resourceType: 'MeasureReport',
+  type: 'individual',
+  status: 'complete',
+  measure: '',
+  period: {
+    start: '',
+    end: ''
+  },
+  text: {
+    div: 'test123',
+    status: 'additional'
+  }
+};
+
+const MOCK_BUNDLE: fhir4.Bundle = {
+  resourceType: 'Bundle',
+  type: 'collection'
+};
+
+describe('PopulationCalculation', () => {
+  it('should not render Calculate Population Results button by default', () => {
+    render(
+      mantineRecoilWrap(
+        <>
+          <PopulationCalculation />
+        </>
+      )
+    );
+
+    const calculateButton = screen.queryByTestId('calculate-all-button');
```

Comment:
When I try `queryByRole` when I'm testing for something that is not in the document, I get the following: 

"TestingLibraryElementError: Unable to find an accessible element with the role "button" and name "Calculate Population Results" There are no accessible roles. But there might be some inaccessible roles. If you wish to access them, then set the `hidden` option to `true`."

I don't have a lot of experience with frontend testing, so I used other `not.toBeInTheDocument()` unit tests as precedent, and they all seem to use `queryByTestId()`. Maybe it worked for your unit test because you did have a button present, but not with the specific name (since yours changes from "show" to "hide")? Not really sure